src/content/docs/architecture.mdx

```diff
@@ -139,51 +141,53 @@ those changes required forking Flutter.
 
 We had to fork 4 Flutter and Dart repositories to make Shorebird work:
 
+#### `flutter/buildroot` "The Buildroot"
+
+The `buildroot` repo contains the build scripts that are used to build the
+Flutter engine for various platforms. It's separate from `flutter/engine` in
+order to share code and configuration with the Fuchsia build system.
+
+We forked this code to make one small modification to the build setup to
+allow exposing our updater symbols from the Flutter engine up to
+`package:shorebird_code_push` and to include `libunwind` since the Rust code
+requires it for backtraces.
+
+https://github.com/shorebirdtech/updater
+is a Rust library which we link into the engine. The way we do that is via
+a C-API on a static library (`libupdater.a`). The default flags for linking
+for the Flutter engine hide all symbols from linked static libraries. We
+need to be able to expose the shorebird\_\* symbols from `libupdater.a` up through
+FFI to the Dart code. We did that by making one change to buildroot and then
+a second change to the engine to place the symbols on the allow-list.
+
+https://github.com/flutter/buildroot/compare/master...shorebirdtech:buildroot:shorebird/dev
```

Comment:
```suggestion
https://github.com/shorebirdtech/updater
is a Rust library which is linked into the Flutter engine via
a static library (`libupdater.a`) that exposes a C API. 

Because the default Flutter engine linking flags
hide all symbols from linked static libraries, and because we
need to be able to expose the `shorebird\_\*` symbols from `libupdater.a` to the Dart code, we made one change to buildroot and then
a second change to the engine to place the symbols on the allow-list.

https://github.com/flutter/buildroot/compare/master...shorebirdtech:buildroot:shorebird/dev
```

Reply:
What is the one change made to buildroot? Would be good to link to a PR/commit or describe it briefly.